old/AbstractBulbCommand.java

```diff
@@ -0,0 +1,131 @@
+package jlifx.commandline;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.net.InetAddress;
+import java.util.Collection;
+import java.util.Collections;
+
+import jlifx.bulb.Bulb;
+import jlifx.bulb.DiscoveryService;
+import jlifx.bulb.GatewayBulb;
+import jlifx.bulb.IBulb;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+public abstract class AbstractBulbCommand implements CommandLineCommand {
+    private boolean interrupted = false;
```

Comment:
\[Checkstyle\] ERROR: Variable 'interrupted' explicitly initialized to 'false' \(default value for its type\)\.